### PR TITLE
Fix pubsub example

### PR DESCRIPTION
**Pull request by Tbaut**
- Fix https://github.com/paritytech/wiki/issues/112

### CODE REVIEW

**Comment by Tbaut** [comment]
ping @folsen

File: JSONRPC-Eth-Pub-Sub-Module.md [hunk 1]
```diff
@@ -51,15 +77,16 @@ params: [
 
 Request
 ```bash
-curl --data '{"method":"eth_subscribe","params":["newHeads",{"fromBlock":"latest","toBlock":"latest"}],"id":1,"jsonrpc":"2.0"}' -H "Content-Type: application/json" -X POST localhost:8545
+wscat -c localhost:8546
```
Comment (folsen) [inline JSONRPC-Eth-Pub-Sub-Module.md]: wscat is not a standard tool, so maybe have a little preface saying what it is and a link to where you can find it

File: JSONRPC-Parity-Pub-Sub-module.md [hunk 2]
```diff
@@ -47,15 +47,23 @@ params: [
...
 Response
 ```js
 {
-  "id": 1,
-  "jsonrpc": "2.0",
-  "result": "0x416d77337e24399d"
+    "jsonrpc": "2.0",
+    "result": "0x070fa1c4d1b3fd81",
+    "id": 1
+} {
```
Comment (folsen) [inline JSONRPC-Parity-Pub-Sub-module.md]: Are these two separate responses or what? the formatting looks a bit odd here, like the first response is a confirmation thing and the second is a new subscription or something?

File: JSONRPC-Eth-Pub-Sub-Module.md (hunk 1 above)
Comment (Tbaut) [reply JSONRPC-Eth-Pub-Sub-Module.md]: fixed

File: JSONRPC-Parity-Pub-Sub-module.md (hunk 2 above)
Comment (Tbaut) [reply JSONRPC-Parity-Pub-Sub-module.md]: Totally makes sense, the second was a notification. I updated the notification example above and removed it from here.

**Comment by Tbaut** [comment]
Grumble addressed.

**Pull request merged**